video/qt_decoder.h

```diff
@@ -206,6 +212,10 @@ class QuickTimeDecoder : public VideoDecoder, public Audio::QuickTimeAudioDecode
 		kUpdateModeOffscreenOnly,
 		kUpdateModeFromOffscreen,
 		kUpdateModeDirectToScreen,
+
+		kScaleFactorOne = 1,
+		kScaleFactorTwo = 2,
+		kScaleFactorThree = 3,
```

Comment:
We were discussing that possible difference between "low quality" antialasing and high quality could be with linear/gaussian interpolation. How is this approach of 2x/3x is compared to this and the original?